buildSrc/.gitignore

```diff
@@ -0,0 +1 @@
+build-bootstrap/
```

Comment:
I kind of like having one at the root with the paths to things in it rather than a bunch of them. I'm somewhat indifferent, but it is nice to have one file to look at when I want to figure out why files aren't going into scm.